.github/workflows/release.yml

```diff
@@ -74,9 +74,10 @@ jobs:
     - name: Create Pull Request
       uses: peter-evans/create-pull-request@v3
       with:
-        commit-message: "chore: release bump [skip actions]"
+        commit-message: "chore: version bump"
         delete-branch: true
         title: 'chore: update cli package version'
-        body: 'Updates the cli version details to align with pyi release'
+        body: 'Updates the cli version details to align with pypi release'
         labels: 'chore'
         base: 'master'
+        branch: create-pull-request/relase_bump
```

Comment:
This looks like a typo but I'm not sure. `release_bump` maybe?

Reply:
```suggestion
        branch: create-pull-request/release_bump
```